src/scene/materials/standard-material-parameters.js

```diff
@@ -45,6 +45,8 @@ const standardMaterialParameterTypes = {
     specularMapTiling: 'vec2',
     specularMapOffset: 'vec2',
     specularMapRotation: 'number',
+    specularityFactor: 'number',
+    specularityFactorMap: 'texture',
```

Comment:
The other map members are also expected by addMap in `standard.js`:
```suggestion
    specularityFactorMap: 'texture',
    specularityFactorMapChannel: 'string',
    specularityFactorMapUv: 'number',
    specularityFactorMapTiling: 'vec2',
    specularityFactorMapOffset: 'vec2',
    specularityFactorMapRotation: 'number',
```